TundraTD/Assets/Scripts/Level/LaunchWaveButton.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using UnityEngine;
+
+namespace Level
+{
+    public class LaunchWaveButton : MonoBehaviour
+    {
+        [SerializeField] private Canvas magicUI;
+        [SerializeField] private LevelCornerman levelCornerman;
+
+        private void Start()
+        {
+            if (magicUI is null || levelCornerman is null)
+                throw new Exception("magicUI or levelCorner man were not assigned");
```

Comment:
Please, specify the exception with the more concrete type
```suggestion
                throw new ArgumentNullException("magicUI or levelCorner man were not assigned");
```